src/main/java/com/harrison/cb/CouchbaseClient.java

```diff
@@ -46,6 +46,13 @@
 	    bucket.upsert(JsonDocument.create("u:king_arthur", arthur));
 	    bucket.upsert(JsonDocument.create("u:king_harrison", harrison));	
 	    
+	    if (1 == 1) {
+	        try {
+	            throw new RuntimeException();
+	        } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-245-133-150.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-245-133-150.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS00108)